src/main/resources/arguments.json

```diff
@@ -336,5 +328,39 @@
         }
       }
     }
+  },
+  "purge": {
+    "contains": {
+      "_type": "command"
+    },
+    "user": {
+      "_type": "command"
+    },
+    "attachments": {
+      "_type": "command"
+    },
+    "bot": {
+      "_type": "command"
+    },
+    "link": {
+      "_type": "command"
+    }
+  },
+  "purge": {
+    "contains": {
+      "_type": "command"
+    },
+    "user": {
+      "_type": "command"
+    },
+    "attachments": {
+      "_type": "command"
+    },
+    "bot": {
+      "_type": "command"
+    },
+    "link": {
+      "_type": "command"
+    }
```

Comment:
I can see you still don't understand how this file works
First off items with `*` after them mean they don't show up in the usage
so for example
```
"purge":
    "command": {
        ect...
    }
}
```
will show up in the usage as
`;purge` - stuff
`'purge command` - stuff
also you didn't implement the actual arguments that users should use.
for example your purge image should look like
```
"purge*":
    "image*": {
        "amount": {
            "_type": "required"
        }
    }
}
```

Reply:
Okay.